net-analyzer/tcpdump/tcpdump-4.99.3.recipe

```diff
@@ -79,6 +80,11 @@ INSTALL()
 	make install
 	strip $binDir/tcpdump
 	rm $binDir/tcpdump.$portVersion
+
+	if [ $effectiveTargetArchitecture = x86 ]; then
```

Comment:
Should be done with:
`runConfigure --omit-dirs binDir ./configure --bindir="$prefix/bin"`

Reply:
done

Reply:
Or with $commandBinDir as it has been done for a while now: https://github.com/Begasus/haikuports/wiki/Useful-variables-for-recipes#commandbindir-and-commandsuffix